runtime/doc/insert.txt

```diff
@@ -1183,6 +1183,9 @@ items:
 	user_data	custom data which is associated with the item and
 			available in |v:completed_item|; it can be any type;
 			defaults to an empty string
+	extrahl		allows specifying extra highlight group names to apply
```

Comment:
The "extra" name is a bit confusing as there are also `PmenuExtra` and `PmenuExtraSel` highlight groups, which are unrelated to this.

Reply:
makes sense. any good suggestions?

Reply:
`hl_name` or `hl_group` ?

Reply:
purpose is to add cterm/gui attributes on the original basis.  meaning of hl_name hl_group is unclear

Reply:
Well, should it should provide attributes or it should be a group name, with the applied highlighting attributes applied on top of it? The documentation above talks about `extra highlight group names to apply`, so `hl-group` sounds like a reasonable choice for a highlighting group, no?

Reply:
That is fine. I don't remember that we need to specify additional highlighting attributes directly. for all kind of functions we assume a proper highlighting group must be defined beforehand and then only this highlighting group is used.  Please adjust the documentation, that the *specified highlighting group*  is combined together with `PmenuSel`/`Pmenu`. Would also be good which highlighting group has priority.